agroal-api/src/main/java/io/agroal/api/configuration/AgroalConnectionPoolConfiguration.java

```diff
@@ -206,8 +207,8 @@ static ConnectionValidator defaultValidator() {
                 @Override
                 public boolean isValid(Connection connection) {
                     try {
-                        return connection.isValid( 0 );
```

Comment:
please respect the code style (space after `(` and before `)`). it's not enforced, but a good practice to keep.

Reply:
That was formatted by the IDE, I'll change it